server/a_socials/settings/base.py

```diff
@@ -23,6 +23,7 @@
   )
 )
 
+TEST_RUNNER = 'snapshottest.django.TestRunner'
```

Comment:
Well-done @parseendavid  but please let's have this in our dev settings since base settings are for both production and development.

Reply:
I will 👍